e2e-playwright/.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
+  "root": true,
+  "extends": [
+    "plugin:prettier/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:playwright/playwright-test"
+  ],
+  "plugins": ["prettier", "import"],
+  "rules": {
+    "no-console": "warn",
+    "import/no-default-export": "error",
+    "import/order": [
+      "error",
+      {
+        "groups": [
+          ["external", "builtin"],
+          ["parent", "internal"],
+          ["index", "sibling"]
+        ],
+        "newlines-between": "always"
+      }
+    ],
+    "prettier/prettier": [
+      "error",
+      {
+        "printWidth": 120,
+        "tabWidth": 2,
+        "singleQuote": true,
+        "trailingComma": "all",
+        "proseWrap": "always"
+      }
```

Comment:
some time ago we were changing eslint settings and I remember we removed the 2nd item in this rule (the config object), so eslint uses the config from .prettierrc, you could do the same here. Probably you will have to manually pass the path to .prettierrc in this case.

I am not sure but I think I saw the option to pass path to .prettierrc file 🙂

Reply:
Good idea, when i started this task i used old version of our starter, where prettierrc config was declarated insite eslint config. Happy that it's separated now :D 
I removed this object from the config, prettier still works in e2e-playwright folder (it react to config changes) without providing any path. :)